File: 157-markdown-link-parser/solve.js
function parseLink(markdown) {
		const startTitle = markdown.indexOf("[")+1;
		const endTitle = markdown.indexOf("]");
		const startLink = markdown.indexOf("(")+1;
		const endLink = markdown.indexOf(")");

  // return markdown;
		return `<a href="${markdown.slice(startLink, endLink)}">${markdown.slice(startTitle, endTitle)}</a>`
}

parseLink("[freeCodeCamp](https://freecodecamp.org/)")

File: 157-markdown-link-parser/solve.py
def parse_link(markdown):
    start_title = markdown.index('[') + 1
    end_title = markdown.index(']')
    start_link = markdown.index('(') + 1
    end_link = markdown.index(')')

    return f'<a href="{markdown[start_link:end_link]}">{markdown[start_title:end_title]}</a>'

#result = parse_link("[freeCodeCamp](https://freecodecamp.org/)")
#print(result)
